### PR TITLE
Issue 161: Add scripts for reading and writing MP4 files from/to Pravega

### LINKED ISSUE #161: Add scripts for reading and writing MP4 files from/to Pravega
**Problem description**
It would be handy to have GStreamer scripts to write and read MP4 files to/from Pravega.

### PR DESCRIPTION
**Pull request by RaulGracia**
**Change log description**
Added scripts to read/write MP4 files from/to Pravega.

**Purpose of the change**
Fixes #161.

**How to verify it**
Run Pravega in standalone mode and write and read an MP4 file from Pravega.

### CODE REVIEW

File: scripts/mp4-file-to-pravega.sh [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+! queue \
+! timestampcvt input-timestamp-mode=start-at-current-time \
+! mp4mux streamable=true fragment-duration=1 \
+! fragmp4pay \
+! pravegasink \
+  stream=${PRAVEGA_SCOPE}/${PRAVEGA_STREAM} \
+  allow-create-scope=${ALLOW_CREATE_SCOPE} \
+  controller=${PRAVEGA_CONTROLLER} \
+  keycloak-file=\"${KEYCLOAK_SERVICE_ACCOUNT_FILE}\" \
+  seal=false sync=false timestamp-mode=tai
```
Comment (vangork) [inline scripts/mp4-file-to-pravega.sh:47]: Please end the file with a LF.

File: scripts/mp4-pravega-to-screen.sh [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  controller=${PRAVEGA_CONTROLLER_URI} \
+  keycloak-file=\"${KEYCLOAK_SERVICE_ACCOUNT_FILE}\" \
+  stream=${PRAVEGA_SCOPE}/${PRAVEGA_STREAM} \
+  $* \
+! qtdemux name=qtdemux \
+! h264parse name=h264parse \
+! video/x-h264,alignment=au \
+! avdec_h264 \
+! videoconvert \
+! autovideosink sync=true
```
Comment (vangork) [inline scripts/mp4-pravega-to-screen.sh:40]: Please end the file with a LF.

**Review by vangork — CHANGES_REQUESTED** [review]

File: scripts/mp4-file-to-pravega.sh (hunk 1 above)
Comment (RaulGracia) [reply scripts/mp4-file-to-pravega.sh]: Added blank line at the end.

File: scripts/mp4-pravega-to-screen.sh (hunk 2 above)
Comment (RaulGracia) [reply scripts/mp4-pravega-to-screen.sh:40]: Added blank line at the end.

**Comment by RaulGracia** [comment]
@vangork thanks for the feedback. As we discussed, I have added 2 versions of the script that writes an MP4 file to Pravega, depending on the mux option. Thanks.

**Comment by vangork** [comment]
Looks good!

**Comment by vangork** [comment]
Please sign all the commits and merge.

**Pull request merged**